docs/Release_notes.md

```diff
@@ -1,5 +1,10 @@
 V2X-Hub Release Notes
 ----------------------------
+Version 7.2.2, released May 9th, 2022
+--------------------------------------------------------
```

Comment:
V2xhub can control the number of times each TCM being repeatedly broadcast upon receiving TCMs from carma-cloud. After it repeatedly broadcast the TCMs, it waits for a configured time duration before it sends a time out message to carma-cloud notifying that no response from CMV for TCMs. If V2xHub receives an acknowledgement from CMV, it will stop broadcasting TCMs and display the acknowledgement on the v2xhub admin user interface.